### ISSUE TITLE
convert degree days to days if possible

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/ospree#72]
Revisit each paper with a degree day respvar and see if we can back convert to days (e.g., see basler12). And note HOW they calculated degree days (e.g., a lot seemed to follow heide93).

**Comment by lizzieinvancouver** [comment]
try ...
lookatme <- subset(d, respvar.simple=="thermaltime")
unique(lookatme$datasetID)

... returns:
"basler12"     "ghelardini10" "heide93"      "karlsson03"   "laube14a"     "skuterud94" 

decide where to make changes (in clean_respvar.R?)

**Comment by lizzieinvancouver** [comment]
basler12 - no go "The degree-days at bud burst were calculated from the reconstructed daily mean temperatures at the sampling site (from 1 January until sampling) and the daily mean treatment temperatures (from sampling until bud burst), using 0 ◦ C as base temperature for degree-day accumulation, as recommended by Heide (1993a)"

**Comment by lizzieinvancouver** [comment]
ghelardini10" -- maybe possible? "which were terminated after accumulation of 1050 day degrees (dd) >0 °C, i.e., after 50, 40 and 75 days at 21 (Ex- periment 1), 26 and 14 °C (Experiment 2), respectively."

**Comment by lizzieinvancouver** [comment]
laube14a seems tricky, need to check carefully

**Comment by cchambe12** [comment]
working on ghelardini10 right now, which seems pretty straightforward. I am making figures to first show the change before committing in ospree, want to confirm changes with group first!

**Comment by cchambe12** [comment]
looks like heide93 is like ghelardini and karlsson03 is also similar but response and response.time columns are switched. need help with code to fix these three studies. skuterud94 does not specify. I need more time to work on laube14a

**Comment by cchambe12** [comment]
changed ghelardini10, heide93, karlsson03 to daystobudburst. Unable to fix basler12, skuterud94, and laube14a

**Comment by lizzieinvancouver** [comment]
I have a few queries!

(1) If we are not changing karlsson03 should we change the respvar.simple (the code does but that seems bad).
(2) Also, I think we should change the respvar also for the other two studies that we are changing. For example:
d<-within(d, respvar.simple[datasetID=="ghelardini10" &
    respvar=="thermaltimetobudburst"]<-"daystobudburst.converted")
(3) We should add a note about where in each paper the thermal time conversion equation is given (is it really just mean temp each day? No summations or such?)
(4) I noticed in ghelardini10 really high response values (700-1000)... why?

**Comment by cchambe12** [comment]
Okay... I hope this issue is actually closed this time...

So basler12 was already entered in another way so keeping thermaltime values. Same as ghelardini10. To avoid redundancies, I left these alone.

heide93 I was able to fix figure 3 & 4 
karlsson03 I was able to fix figure 1

I have added equations in the clean_thermaltimetodays.R and in thermaltimeconvert_README.rtf

Almost closed. Need to work on swarts81
